File: Codeforces/C++/problemset/Stones on the Table.cpp
// Problem Link: https://codeforces.com/problemset/problem/266/A

#include <iostream>
using namespace std;
int main()
{
    long long n,x = 0;
    cin >> n;
    char *s = new char[n];
    for(long long i = 0; i < n; i++)
    {
        cin >> s[i];
        if(i > 0 && s[i] == s[i-1]) x++;
    }
    cout << x;
    delete [] s;
    return 0;
}

File: Codeforces/Kotlin/src/main/kotlin/problems/StonesOnTheTable.kt
package problems.problems

/* Problem Link: https://codeforces.com/problemset/problem/266/A */

fun main() {
    val n = readln().toInt()
    val stones = readln()
    var number = 0

    for (i in 0..stones.length - 2) {
        if(stones[i] == stones[i+1]) number ++
    }

    print(number)
}